HealthVault/Classes/Models/Categories/NSArray+Utils.h

```diff
@@ -41,6 +41,8 @@
 
 - (NSArray *)arrayByRemovingObject:(id)object;
```

Comment:
This whole file needs to be cleaned up. Most of the methods are not used anywhere - They should be deleted. Also let's get rid of the convenience methods like arrayByRemovingObject: We should just do this with a mutable array where needed.